### ISSUE TITLE
Recent Files menu does not disappear when it should

### ISSUE THREAD

**Issue by tbethe** [ddvk/remarkable-hacks#256]
After the update to 2.6, when opening the recent files menu, there are only 2 ways to get rid of it. Click on the R button in the toolbar, or click on one of the recent files, but before the update, I could just click anywhere and the menu would disappear.

Reproduce: 
(1) Open the recents menu > Click anywhere on the screen (except the R button/on the menu itself) > menu doesn't disappear.
(2) Open recents menu > Close the file (battery icon > click on X) > recents menu is still on the screen while on the main menu of xochitl. 

Expected behaviour: 
(1) When clicking anywhere outside the recents menu, the menu should disappear. 
(2) When leaving the file, the recents menu should also disappear.

Device: reMarkable 2.
reMarkable Version: 2.6
remarkable-hacks: patch for 2.6

**Comment by ddvk** [comment]
you can swipe woth 2 finger up beside the me u

**Comment by tbethe** [comment]
That's true, but the menu still persists when going to the main menu, and then two finger swipe does not work anymore. 

Perhaps my memory is fooling me, but I think it was different before the update.

It's not a big deal, regardless.
